tutorials/circuits/3_summary_of_quantum_operations.ipynb

```diff
@@ -1592,7 +1592,7 @@
    "source": [
     "#### Controlled $Y$ gate\n",
     "\n",
-    "Apply the $Y$ gate to the target qubit if the control qubit is the MSB\n",
+    "Apply the $CY$ gate to the target qubit if the control qubit is the MSB\n",
```

Comment:
```suggestion
    "Apply the $Y$ gate to the target qubit if the control qubit is the MSB\n",
```
The original text was correct. Applying $CY$ gate is means applying $Y$ gate to the target qubit when the control qubit is 1. The same applies to the rest of the changes.